mender-update/daemon/state_machine/state_machine.cpp

```diff
@@ -291,6 +298,8 @@ error::Error StateMachine::Run() {
 		return err;
 	}
 
+	log::Info("Running Mender client " + conf::kMenderVersion);
```

Comment:
Q: Wrong commit?

Reply:
No, it's correct. We want each deployment log, as well as after every reboot, to start with the version, for debugging reasons in the field, like we do [in the Golang client](https://github.com/mendersoftware/mender/blob/3.5.1/app/deployment_logger.go#L144).